bin/gcs_cli.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+# Manipulate emulated GCS storage.
+
+# Usage: ./bin/gcs_cli.py CMD
+
+import os
+
+import click
+
+from google.auth.credentials import AnonymousCredentials
+from google.cloud import storage
+from google.cloud.exceptions import NotFound
+
+
+def get_endpoint_url(bucket_name):
+    base_url = os.environ["STORAGE_EMULATOR_HOST"]
+    return f"{base_url}/storage/v1/b/{bucket_name}"
+
+
+def get_client():
+    return storage.Client(credentials=AnonymousCredentials(), project="test")
```

Comment:
```suggestion
        project_id = os.environ["STORAGE_PROJECT_ID"]
        return storage.Client(credentials=AnonymousCredentials(), project=project_id)
```

The project id should be configured by an env var.